### DISCUSSION TITLE
PeoplePicker in Fluent 2 (v9)

### DISCUSSION THREAD

**Discussion by rhalaly** [microsoft/fluentui#30747] [Q&A] [answered]
I can't see the `PeoplePicker` component in Fluent 2 (v9), and no migration comment about this component.

How can I use the `PeoplePicker` in Fluent 2 (v9)?

**Comment by pradeept95** [comment, marked as answer]
Hi @rhalaly,
PeoplePicker/TagPicker is currently is in progress. You can find more details:

https://github.com/microsoft/fluentui/issues/26652
https://github.com/microsoft/fluentui/pull/30306
